### PR TITLE
[BULL-33371] Provide a Getting Started guide

### PR DESCRIPTION
**Pull request by danielkoh-88**

### CODE REVIEW

**Review by kar-arindam — APPROVED** [review]
lgtm

File: README.md [hunk 1]
```diff
@@ -1,40 +1,130 @@
 
-Python 3 Trading Scripts
---------
---------
+Bullish API Examples
+===================
 
-This repository contains Python 3 API trading scripts for [Bullish](https://exchange.bullish.com/) cryptocurrency 
-exchange. This set of scripts can be used out of the box. API traders can use these scripts as a starting point.
+This repository contain code examples to help developers get started with using the [Bullish cryptocurrency exchange](https://exchange.bullish.com/), via a programmatic interface.
```
Comment (dexterhuang-blockone) [inline README.md]: contain -> contains

File: README.md [hunk 2]
```diff
@@ -1,40 +1,130 @@
...
+print(f"Response HTTP Status: {response.status_code}")
+markets = response.json()
+pprint.pp(markets) #pretty print
 ```
-pip3 install -r requirements.txt
+
+### Prerequisite to using private APIs - the JWT token
+Important details about [how to start sending authenticated requests here](https://api.exchange.bullish.com/docs/api/rest/trading-api/v2/#overview--how-to-send-authenticated-requests). Some sample code on how to actually generate a JWT token:
+- ECDSA example: [generate_jwt_v2.py](generate_jwt_v2.py)
+- HMAC example: [generate_jwt_hmac.py](generate_jwt_hmac.py)
```
Comment (dexterhuang-blockone) [inline README.md]: iirc, we wanted to remove these duplicated ones at the top level directory and use the ones inside the respective `/ecdsa`, `/hmac` folders.

Comment (dexterhuang-blockone) [reply README.md]: Same for the other links in this README.

File: README.md [hunk 3]
```diff
@@ -1,40 +1,130 @@
...
+There are module dependencies that the code samples depend on. It is recommended the setup is done in a Python virtual environment, which can be initialised with:
+```bash
+python3 -m venv .
+source ./bin/activate
+```
+Following which, we install the dependencies in [requirements.txt](requirements.txt)
+```bash
+pip3 install -r requirements.txt
+```
+In order to actually run the samples that involve calling private APIs, __credentials such as API keys and secrets need to be provided__. Our samples read these from the [.env file](.env) which look like:
```
Comment (dexterhuang-blockone) [inline README.md]: like -> likes

File: README.md (hunk 2 above)
Comment (danielkoh-88) [reply README.md]: Fixed for most, except where the ones in the main folder is more recent. I plan to reorganise all the samples separately, so will update again then

**Review by dexterhuang-blockone — APPROVED** [review]

**Pull request merged**